src/components/Search/DateTimeDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import { Tooltip } from '@mui/material';
+
+const DateTimeDisplay = ({ itemDate }: { itemDate: Date }) => {
+  // Format for local time
+  const localDateTime = itemDate?.toLocaleString('en-US', {
+    year: '2-digit',
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: false,
+    timeZoneName: 'short',
+  });
```

Comment:
instead of calling `toLocalString` repeatedly, which can be a performance issue, you can create a Intl.DateTimeFormat outside of the component.

Outside of the component, you'd do something like this:
```js
const localDateFormat = new Intl.DateTimeFormat('en-US', { ... });
const utcDateFormat = new Intl.DateTimeFormat('en-US', { ... });
``

Then inside the component, you can use it this way:
```js
const localDateTime = localDateFormat.format(itemDate);
const utcDateTime = utcDateFormat.format(itemDate);
```

This way, the expensive initialization of the date-time locale formatting is done just once at startup.

Reply:
That makes perfect sense!
And I have tweaked it